server/realtime/interactor.go

```diff
@@ -0,0 +1,36 @@
+package realtime
+
+import (
+	"github.com/nchaloult/codenames/model"
+)
+
+// Interactor facilitates events through a Websocket connection with clients.
+// Each Interactor instance is associated with a Game instance; they have a 1-1
+// relationship. Whenever a client performs an action that triggers an event,
+// an Interactor instance processes that
+// Players/connected clients.
```

Comment:
This section of the `Interactor` struct's documentation looks like it's lingering around from a previous "version" of this writeup. Is that true, or does this belong?